packages/tweetbox-app/src/views/Dashboard.vue

```diff
@@ -111,8 +120,20 @@ export default {
         async getTweetWithID() {
 
         },
-        async deleteTweetWithID() {
-
+        async deleteTweetWithId() {
+            try{
+                console.log("myID is: " + this.deleteId);
+                if (this.deleteId !== ''){
+                    const response = await deleteTweet(this.deleteId);
+                    console.log(response);
+                    swal('Deleted!', 'tweet delete sucessfully.', 'delete');
```

Comment:
This has to be -> swal('Deleted!', 'tweet deleted sucessfully.', 'success');
Please make the change.